docs/introduction/Base-App.rst

```diff
@@ -421,6 +422,19 @@ Each pair on the passed list represents a path to be disabled (the key is the ba
 		.annotatedWith(Names.named("RejectedApiPaths"))
 		.toInstance(...); 
 
+- Sidechain Application stopper
+It is a customized class instance which implements the public interface 'SidechainAppStopper' and must provide an
+implementation of the method 'void stopAll()'. Such a method is called by the SDK when the node stop procedure is initiated.
+Such a procedure can be explicitly triggered via the API 'node/stop' or can be triggered when the JVM is shutting down,
+for instance when a SIGINT is received.
```

Comment:
Just to avoid repetition of "for instance" in the two senteces, you could use "for example". 

`......Such a procedure can be explicitly triggered via the API 'node/stop' or can be triggered when the JVM is shutting down,
for example when a SIGINT is received.
In the custom implementation for instance, custom storages should be closed or..........`